src/processors/data-processor.js

```diff
@@ -422,6 +422,11 @@ export function processGeojson(rawData) {
   // getting all feature fields
   const allData = normalizedGeojson.features.reduce((accu, f, i) => {
     if (f.geometry) {
+      Object.keys(f.properties).forEach(key => {
```

Comment:
This will error if `properties` doesn't exist or not an object. I would do a `typeof f.properties === 'object' && f.properties !== null) check

Reply:
@heshan0131 Thanks for your review, I add this validation and update this PR